changelog/fix-no-side-spacing-mobile-lm-quiz

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Lateral spacing not applied in the mobile view of quiz in learning mode
```

Comment:
Is this something that currently exists in the 4.18 version? If it's something we introduced in 4.19, then we probably don't need the change log in this PR or the Pro one.

Reply:
> Is this something that currently exists in the 4.18 version?

It was introduced in 4.19, doesn't exist in 4.18. I've removed the changelog 👍 Thanks! (Also dismissed the review when I removed the changelog, sorry 😄 )